examples/quantum.py

```diff
@@ -0,0 +1,463 @@
+'''
+This file contains
+* MicroQiskit (https://github.com/qiskit-community/MicroQiskit/blob/master/README.md)
+* `make_line` from QuantumBlur (https://github.com/qiskit-community/QuantumBlur/blob/master/README.md)
+Both are licensed under Apache 2.0, copyright of IBM Research and written by James R. Wootton (@quantumjim)
+Everything else here is under the same license as the rest of the repo.
+
+Two optional inputs can be supplied when running the file.
+The first is a 1 or 0, to determine whether to implement a quantum interference pattern or just use random bit flips.
+The second is the maxmimum pixel brightness. By default, quantum is on and the max brightness is 128.
+The third is whethe
```

Comment:
I think there's a bit of this comment missing?

Reply:
Yep, removed!

Reply:
Sweet, thanks.